lib/kafka_ex/server.ex

```diff
@@ -127,11 +127,14 @@ defmodule KafkaEx.Server do
   end
 
   def handle_call({:create_stream, handler}, _from, state) do
-    state = unless state.event_pid && Process.alive?(state.event_pid) do
+    if state.event_pid && Process.alive?(state.event_pid) do
+      info = Process.info(self)
+      Logger.log(:warn, "#{info[:registered_name]} already streaming")
```

Comment:
@bjhaid i updated the log msg to include the name of the process, not the handler, as it seems to be more related than the handler involved. does that make sense?